src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -847,6 +847,12 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 				)
 
 			},
+			didTapInfoProfile: {
+				// Todo: Navigate to profile info
```

Comment:
Are these going to be done in another ticket ?
```suggestion
				// Todo: Navigate to profile info
```

Reply:
Yes